src/main/scala/dal/table/blockTable.scala

```diff
@@ -0,0 +1,13 @@
+package dal.table
+
+import sqlest._
+import org.joda.time.LocalDate
+
+class BlockTable(alias: Option[String]) extends Table(block) {
+  val blockID = column[Int]("block_id")
+  val description = column[String]("description")
+  val blockTimeStamp = column[Option[LocalDate]]("time_stamp")
```

Comment:
Will LocalDate work for a timestamp column? I had a look in the AS code and there is a DateTime type which they use for timestamp cols.

Reply:
Also I would argue that this one isn't an option.

Reply:
Not option on table, but may need to be option on the domain class for uncommitted blocks

Reply:
Ah I was blindly copying other code with that Option decision. Agree with your points